lib/KTable/index.vue

```diff
@@ -0,0 +1,480 @@
+<template>
+
+  <div ref="tableWrapper" class="k-table-wrapper">
+    <table class="k-table" role="grid">
+      <caption v-if="caption">
+        {{ caption }}
+      </caption>
+      <thead>
+        <tr>
+          <th
+            v-for="(header, index) in headers"
+            :key="index"
+            tabindex="0"
+            :aria-sort="sortable && header.dataType !== DATA_TYPE_OTHERS ? getAriaSort(index) : null"
+            :class="{
+              [$computedClass(coreOutlineFocus)]: true,
```

Comment:
I think this could be simplified a bit?

```suggestion
              $computedClass(coreOutlineFocus): true,
```

Reply:
Gives me this error when doing so.
[{
	"resource": "/home/anoushka/kolibri-design-system/lib/KTable/index.vue",
	"owner": "_generated_diagnostic_collection_name_#0",
	"code": "8010",
	"severity": 8,
	"message": "Type annotations can only be used in TypeScript files.",
	"source": "ts",
	"startLineNumber": 16,
	"startColumn": 49,
	"endLineNumber": 16,
	"endColumn": 53
}]

Reply:
My bad, I didn't notice this is attribute of a wrapping object